### ISSUE TITLE
notifier.exe  crashing

### ISSUE THREAD

**Issue by heml0ck** [wokhansoft/WFN#13]
crashes every time without running as admin manually.
anytime a connection invokes, it crashes with System.UnauthorizedAccess

started happening after updating, not exactly sure which version from, believe it was the last one on codeplex before the migration announcement, updated to latest beta 1 and 2, tried to revert and still the same.

**Comment by AtlasHackert** [comment]
Can you post (the relevant part of) the log-file (in the WFN directory, called "Wokhan.WindowsFirewallNotifier.Common.log") so hopefully I can see where it's crashing?

**Comment by heml0ck** [comment]
sorry, I replied by e-mail, I could post a vid on youtube if you like

**Comment by heml0ck** [comment]
well I give up, tried a bunch of MS utilities and other forms of uninstalling/reinstalling .net framework

the only other thing i see in the log other than the normal stuff i showed you in the e-mail is

[WARNING] Unable to retrieve the target process SessionID. Process may have already exited.

**Comment by heml0ck** [comment]
well, i just tried 1.8.0 again, and it actually works!

**Comment by AtlasHackert** [comment]
I see you're getting an UnauthorizedException error. What folder did you put WFN into? (It currently can't run properly from folders in which it cannot write, due to the log-file.)

**Comment by heml0ck** [comment]
program files x86, log file is updating properly, just nothing useful.  ive given manual permissions absolutely everywhere too.

**Comment by AtlasHackert** [comment]
Are you sure it's also updated from your user, and not just from SYSTEM? Can you post the log-file here?

**Comment by heml0ck** [comment]
argh, like i said, using 1.8.0 now since it works, dont really want to try again and jinx it, I did reply to the first post via e-mail and posted the log, also, in my 3rd comment here, the only other lines in it were ;
[WARNING] Unable to retrieve the target process SessionID. Process may have already exited.
I did check the folder permissions and my user did have full control, if thats what you mean by 'being able to update from my user'
I may try again in a bit.
